modules/imaging_browser/test/imaging_browserTest.php

```diff
@@ -11,33 +11,1057 @@
+        )->getText();
+        $this->assertContains("Brainbrowser", $BreadCrumbText);
+    }
+*/
+
+    /**
+     * Steps 3 through 7
+     * Visit level feedback
+     * 
+     *
+     * @return void
+    **/
+
+    function testViewSessionVisitLevelFeedback()
+    {
+	// Setting permissions to view all sites to view all datasets
+        $this->setupPermissions(array('imaging_browser_view_allsites'));
+        $this->webDriver->navigate()->refresh();
+        $this->webDriver->get(
+            $this->url . "/imaging_browser/"
+        );
+
+        $NativeLink = $this->webDriver->findElement(
+            WebDriverBy::xPath('//*[@id="lorisworkspace"]/div[2]/div/div/table/tbody/tr/td[12]/a')
+        );
+
+	$NativeLink->click();
+	sleep(5);
+
+        $VisitLevelFeedback = $this->webDriver->findElement(
+            WebDriverBy::xPath('//*[@id="sidebar-content"]/div[1]/a/span/span[2]')
+        );
+
+	$VisitLevelFeedback->click();
+
+
+//	ATEMTPTING TO CHANGE THE WINDOW HANDLES TO POPUP, NO SUCCESS
+//	$handles = $this->webDriver->get(
+//	    $this->window_handles()
+//	);
+//	$last_window = end($handles);
+//	$this->$webDriver->focusWindow($last_window);
+
+//        $PopUpPSCID = $this->webDriver->findElement(
+//            WebDriverBy::xPath('/html/body/div/table/tbody/tr[2]/td')
+//        )->getText();
+//        $this->assertContains("AOL0002", $PopUpPSCID);
+
+
+        $QCStatus = $this->webDriver->findElement(
+            WebDriverBy::xPath('//*[@id="sidebar-content"]/div[2]/div/label')
+        )->getText();
+        $this->assertContains("QC Status", $QCStatus);
+
+        $QCPending = $this->webDriver->findElement(
+            WebDriverBy::xPath('//*[@id="sidebar-content"]/div[2]/label')
+        )->getText();
+        $this->assertContains("QC Pending", $QCPending);
+
+
+	// Check that the QC status are editable with correct permission
+        $this->setupPermissions(array('imaging_browser_view_allsites', 'imaging_browser_qc'));
+        $this->webDriver->navigate()->refresh();
+
+        $QCPendingNo = $this->webDriver->findElement(
+            WebDriverBy::xPath('//*[@id="sidebar-content"]/div[2]/select/option[2]')
+        )->getText();
+        $this->assertContains("No", $QCPendingNo);
+
+        $QCStatusPass = $this->webDriver->findElement(
+            WebDriverBy::xPath('//*[@id="sidebar-content"]/div[2]/div/select/option[2]')
+        )->getText();
+        $this->assertContains("Pass", $QCStatusPass);
+
+	// Test that we can edit the QC status by changing it from Blank to Pass
+
+	// Send option Pass (second option) from dropdown menu,
+	// Click save,
+	// Check PASS green flag appears next to file name
+
+
+        $QCStatusSetPass = $this->webDriver->findElement(
+//            WebDriverBy::cssSelector(".col-xs-3 > div:nth-child(1) > div:nth-child(1) > select:nth-child(2) > option:nth-child(2)")
+            WebDriverBy::cssSelector("select.user-success > option:nth-child(1)")
```

Comment:
This css selector is not valid. You could use WebDriverBy::Name('visit_status') and the sendKey() function instead.
ex: 
        $this->webDriver->findElement(
            WebDriverBy::Name('visit_status')
        )->sendKeys("Pass");